TileStache/Goodies/VecTiles/sort.py

```diff
@@ -51,8 +51,17 @@ def _by_transit_score(feature):
     return props.get('mz_transit_score', 0)
 
 
-def _sort_by_transit_score_then_feature_id(features):
+def _by_peak_elevation(feature):
+    wkb, props, fid = feature
+    kind = props.get('kind')
+    if kind != 'peak' and kind != 'volcano':
+        return 0
+    return props.get('elevation', 0)
```

Comment:
Yeah, it should already be a [float in meters](https://github.com/mapzen/TileStache/blob/524-peak-tile-kind-rank/TileStache/Goodies/VecTiles/transform.py#L3173-L3183) from a previous filter step.